x-pack/plugins/fleet/server/services/package_policies/utils.ts

```diff
@@ -28,17 +28,16 @@ import { licenseService } from '../license';
 import { outputService } from '../output';
 import { appContextService } from '../app_context';
 
-export const mapPackagePolicySavedObjectToPackagePolicy = ({
-  id,
-  version,
-  attributes,
-  namespaces,
-}: SavedObject<PackagePolicySOAttributes>): PackagePolicy => {
+export const mapPackagePolicySavedObjectToPackagePolicy = (
+  { id, version, attributes }: SavedObject<PackagePolicySOAttributes>,
+  namespaces?: string[]
+): PackagePolicy => {
+  const { bump_agent_policy_revision: bumpAgentPolicyRevision, ...restAttributes } = attributes;
```

Comment:
`bump_agent_policy_revision` is only added to the SO type, it is removed from `PackagePolicy` to avoid it leaking out in the API responses.